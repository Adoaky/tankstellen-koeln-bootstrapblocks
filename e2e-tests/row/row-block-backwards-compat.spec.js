/**
 * WordPress dependencies
 */
import {
	createNewPost,
	setPostContent,
	getEditedPostContent,
} from '@wordpress/e2e-test-utils';

import rowContent100 from './row-block-content/row-1.0.0'

describe( 'row block backwards compatibility', () => {
	beforeEach( async () => {
		await createNewPost();
	} );

	it( 'Test v1.0.0 row block content', async () => {
		await setPostContent( rowContent100 )

		expect( await page.$( '[data-type="wp-bootstrap-blocks/row"]' ) ).not.toBeNull();
		expect( await page.$$( '[data-type="wp-bootstrap-blocks/column"]' ) ).toHaveLength(2);

		expect( await getEditedPostContent() ).toMatchSnapshot();
	} );
} );
