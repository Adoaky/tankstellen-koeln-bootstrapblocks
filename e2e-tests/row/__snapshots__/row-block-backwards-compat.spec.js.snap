// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`row block backwards compatibility Test v1.0.0 row block content 1`] = `
"<!-- wp:wp-bootstrap-blocks/row -->
<!-- wp:wp-bootstrap-blocks/column {\\"sizeMd\\":6} /-->

<!-- wp:wp-bootstrap-blocks/column {\\"sizeMd\\":6} /-->
<!-- /wp:wp-bootstrap-blocks/row -->"
`;
